### PR TITLE
Allow user attribution on CLI file import

### PR DESCRIPTION
**Pull request by phlax**
Fixes first half of #4142

### CODE REVIEW

**Comment by unho** [comment]
Would be nice if you can add this change to release notes as well. Please squash both commits as well since docs changes are not so extensive.

File: docs/server/commands.rst [hunk 1]
```diff
@@ -563,6 +563,16 @@ counter is older than on Pootle, that is someone has translated while the file
 was offline, then it will be rejected.  Otherwise the translations in the file
 are accepted.
 
+Available options:
+
+:option:`--user`
+  .. versionadded:: 2.7.3
+
+  Import file(s) as given user. If set this option should the username of a
```
Comment (julen) [inline docs/server/commands.rst]: The sentence is incomplete, there seems to be some word missing. I'd consider rewording it as simple as *The user with the provided username must exist.*

File: pootle/apps/import_export/management/commands/import.py [hunk 2]
```diff
@@ -9,19 +9,37 @@
 
 import os
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+from optparse import make_option
 from zipfile import is_zipfile, ZipFile
 
 from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth import get_user_model
```
Comment (julen) [inline pootle/apps/import_export/management/commands/import.py]: Nit: `django.contrib` comes before `django.core`

**Comment by julen** [comment]
Docs are broken on Travis. GTM after fixing that and addressing the comments.

**Pull request merged**